docs/core/diagnostics/FAQ-dumps.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Dumps - .NET FAQ
+description: Frequently Asked Questions about dumps .NET
+ms.date: 09/19/2022
+---
+
+# Frequently Asked Questions Regarding Dumps
+
+## Collecting dumps on macOS and Linux
+
+Collecting a dump requires that the processes in the system have access to call `ptrace` targeting parent processes. The following is a non-exhaustive list of settings that are needed for this to work properly:
+
+- On Linux-based systems, `/proc/sys/kernel` should have `ptrace_scope` set to `1` or `0`. Any value higher than this will require running as root or will block dump collection entirely.
+- For applications running under an Open Container Initiative technology, the `seccomp` profile must allow for calls to `ptrace`. For example, moby+containerd specifies a default [seccomp profile](https://github.com/moby/moby/blob/master/profiles/seccomp/default.json) that allows `ptrace` only if the container host has a kernel version higher than 4.8 or if the `CAP_SYS_PTRACE` capability was specified.
```

Comment:
```suggestion
- For applications running under an Open Container Initiative technology, the `seccomp` profile must allow for calls to `ptrace`. For example, [containerd](https://github.com/moby/containerd) specifies a default [seccomp profile](https://github.com/moby/moby/blob/master/profiles/seccomp/default.json) that allows `ptrace` only if the container host has a kernel version higher than 4.8 or if the `CAP_SYS_PTRACE` capability was specified.
```

Reply:
Is the moby part needed? It just looked a little weird to me.

Reply:
Moby builds on containerd. Moby passes a profile to containerd, so it's a compound tech. I can think of a better way of phrasing this.

Reply:
```suggestion
- For applications running under an Open Container Initiative technology, the `seccomp` profile must allow for calls to `ptrace`. For example, `Docker` uses [containerd](https://github.com/moby/containerd) under the hood as a container runtime. When initializing the runtime, it specified specifies a default [seccomp profile](https://github.com/moby/moby/blob/master/profiles/seccomp/default.json) that allows `ptrace` only if the container host has a kernel version higher than 4.8 or if the `CAP_SYS_PTRACE` capability was specified.
```

Reply:
@gewarren does this look ok without needing to go too much into detail in extraneous techologies?

Reply:
Looks great thanks - I'm just going to make one little edit to remove a duplicated word.